### PR TITLE
Add SetFocusOnEntryCompletedBehavior

### PR DESCRIPTION
**Pull request by bijington**
Documentation added for .NET MAUI Community Toolkit proposals:

- [Proposal] `SetFocusOnEntryCompletedBehavior` <https://github.com/CommunityToolkit/Maui/issues/36>

### CODE REVIEW

**Comment by PRMerger10** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 Changed Files | This PR contains more than 10 changed files.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

File: docs/maui/behaviors/set-focus-when-entry-completed-behavior.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    {
+        var firstName = new Entry
+        {
+            Placeholder = "Entry 1 (Tap `Next` on the keyboard when finished)",
+            ReturnType = ReturnType.Next
+        };
+
+        var lastName = new Entry();
+
+        SetFocusOnEntryCompletedBehavior.NextElement = lastName;
```
Comment (bijington) [inline docs/maui/behaviors/set-focus-when-entry-completed-behavior.md:62]:
```suggestion
        SetFocusOnEntryCompletedBehavior.SetNextElement(firstName, lastName);
```

File: docs/maui/behaviors/set-focus-when-entry-completed-behavior.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+            {
+                new Entry { ReturnType = ReturnType.Next }
+                    .Placeholder("Entry 1 (Tap `Next` on the keyboard when finished)"),
+
+                new Entry()
+                    .Assign(out lastName)
+            }
+        };
+
+        SetFocusOnEntryCompletedBehavior.NextElement = lastName;
```
Comment (bijington) [inline docs/maui/behaviors/set-focus-when-entry-completed-behavior.md:101]:
```suggestion
        SetFocusOnEntryCompletedBehavior.SetNextElement(firstName, lastName);
```

File: docs/maui/behaviors/set-focus-when-entry-completed-behavior.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+{
+    public SetFocusOnEntryCompletedBehaviorPage()
+    {
+        Content = new VerticalStackLayout
+        {
+            Spacing = 12,
+            Children = 
+            {
+                new Entry { ReturnType = ReturnType.Next }
+                    .Placeholder("Entry 1 (Tap `Next` on the keyboard when finished)"),
```
Comment (bijington) [inline docs/maui/behaviors/set-focus-when-entry-completed-behavior.md:94]:
```suggestion
                new Entry { ReturnType = ReturnType.Next }
                    .Assign(out var firstName)
                    .Placeholder("Entry 1 (Tap `Next` on the keyboard when finished)"),
```

**Comment by PRMerger3** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 Changed Files | This PR contains more than 10 changed files.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger6** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 Changed Files | This PR contains more than 10 changed files.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger12** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 Changed Files | This PR contains more than 10 changed files.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger17** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 Changed Files | This PR contains more than 10 changed files.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Review by jfversluis — APPROVED** [review]

**Comment by PRMerger16** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 Changed Files | This PR contains more than 10 changed files.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Pull request merged**